Extension/src/main.ts

```diff
@@ -114,7 +115,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<CppToo
             const config: string = path.join(vscode.workspace.workspaceFolders[i].uri.fsPath, ".vscode/c_cpp_properties.json");
             if (await util.checkFileExists(config)) {
                 const doc: vscode.TextDocument = await vscode.workspace.openTextDocument(config);
-                vscode.languages.setTextDocumentLanguage(doc, "jsonc");
+                await vscode.languages.setTextDocumentLanguage(doc, "jsonc");
```

Comment:
Why is this await and not void?

Reply:
Personal Intuition -- there may be a few things that I marked `await` based on what I saw.  We'll examine some of these tomorrow and I'll adjust based on intent.